packages/language/src/document.js

```diff
@@ -601,8 +660,21 @@ Document.prototype.detectEntities = function(options, callback) {
  *   }
  *
  *   // sentiment = {
- *   //   polarity: 100,
- *   //   magnitude: 40
+ *   //   sentiment: {
```

Comment:
Let's bump the properties in this object up a level so we don't have "sentiment.sentiment"